javascript/ql/src/experimental/Security/CWE-347/examples/results.txt

```diff
@@ -0,0 +1,5 @@
+| col0 |        call         |                                    col2                                    |
++------+---------------------+----------------------------------------------------------------------------+
+|    9 | jwt.ver ... ne"] }) | does not verify the JWT payload with a cryptographic secret or public key. |
+|   10 | jwt.ver ... ne"] }) | does not verify the JWT payload with a cryptographic secret or public key. |
+|   11 | jwt.ver ... ne"] }) | does not verify the JWT payload with a cryptographic secret or public key. |
```

Comment:
This file is not needed, so I suggest you just delete it. 

The comments in `JWTMissingSecretOrPublicKeyVerification.js` are sufficient documentation for what is flagged and what isn't.

Reply:
👍